coalib/parsing/DefaultArgParser.py

```diff
@@ -243,14 +243,25 @@ def default_arg_parser(formatter_class=None):
         '-r', '--relpath', nargs='?', const=True,
         help='return relative paths for files (must be called with --json)')
 
-    outputs_group.add_argument(
+    devtool_exclusive_group = arg_parser.add_mutually_exclusive_group()
+
+    devtool_exclusive_group.add_argument(
```

Comment:
unrelated change, different commit (even PR I would say so we can merge things quicker)

Reply:
Different commit is fine IMO :+1:

Reply:
What the ??

It is not unrelated.  --profile and --debug are mutually exclusive.  It must be done in this commit in order for this commit to be atomic.  Otherwise *this* commit has bugs.  UI bugs.  The kind that coala is riddled with because they are deferred to some other commit instead of UI being considered a vital part of every change.